BaseTools/Source/C/GenFw/Elf64Convert.c

```diff
@@ -534,6 +536,18 @@ WriteSectionRiscV64 (
     mRiscVPass1SymSecIndex = 0;
     break;
 
+  case R_RISCV_GOT_HI20:
+    Value = (Sym->st_value - Rel->r_offset);

```

Comment:
Since the error says:

```
 INFO - Elf64Convert.c(540): error C2220: the following warning is treated as an error
INFO - Elf64Convert.c(540): warning C4244: '=': conversion from 'Elf64_Addr' to 'UINT32', possible loss of data
```

I think an explicit cast to `UINT32` here should fix the VS2019 build.
cc @vlsunil

Reply:
Thanks a lot, @JohnAZoidberg and @lgao4. I was struggling to get the actual error. Let me fix this and send the next version of the patch. It is interesting that GCC could not catch it.